exercises/complex-numbers/complex_numbers_test.py

```diff
@@ -5,73 +5,64 @@
+        second_input = ComplexNumber(0, 2)
         self.assertEqual(first_input.add(second_input).real, 0)
-        self.assertEqual(first_input.add(second_input).imaginary, 4)
+        self.assertEqual(first_input.add(second_input).imaginary, 3)
 
     def test_add_numbers_with_real_and_imaginary_part(self):
         first_input = ComplexNumber(1, 2)
-        second_input = ComplexNumber(4, 6)
-        self.assertEqual(first_input.add(second_input).real, 5)
-        self.assertEqual(first_input.add(second_input).imaginary, 8)
+        second_input = ComplexNumber(3, 4)
+        self.assertEqual(first_input.add(second_input).real, 4)
+        self.assertEqual(first_input.add(second_input).imaginary, 6)
 
     def test_subtract_purely_real_numbers(self):
         first_input = ComplexNumber(1, 0)
-        second_input = ComplexNumber(-1, 0)
-        self.assertEqual(first_input.sub(second_input).real, 2)
+        second_input = ComplexNumber(2, 0)
+        self.assertEqual(first_input.sub(second_input).real, -1)
         self.assertEqual(first_input.sub(second_input).imaginary, 0)
 
+    def test_subtract_purely_imaginary_numbers(self):
+        first_input = ComplexNumber(0, 1)
+        second_input = ComplexNumber(0, 2)
+        self.assertEqual(first_input.sub(second_input).real, 0)
+        self.assertEqual(first_input.sub(second_input).imaginary, -1)
+
     def test_subtract_numbers_with_real_and_imaginary_part(self):
         first_input = ComplexNumber(1, 2)
-        second_input = ComplexNumber(-2, -2)
-        self.assertEqual(first_input.sub(second_input).real, 3)
-        self.assertEqual(first_input.sub(second_input).imaginary, 4)
+        second_input = ComplexNumber(3, 4)
+        self.assertEqual(first_input.sub(second_input).real, -2)
+        self.assertEqual(first_input.sub(second_input).imaginary, -2)
 
     def test_multiply_purely_real_numbers(self):
         first_input = ComplexNumber(1, 0)
         second_input = ComplexNumber(2, 0)
         self.assertEqual(first_input.mul(second_input).real, 2)
         self.assertEqual(first_input.mul(second_input).imaginary, 0)
 
+    def test_multiply_purely_imaginary_numbers(self):
+        first_input = ComplexNumber(0, 1)
+        second_input = ComplexNumber(0, 2)
+        self.assertEqual(first_input.mul(second_input).real, -2)
+        self.assertEqual(first_input.mul(second_input).imaginary, 0)
+
     def test_multiply_numbers_with_real_and_imaginary_part(self):
         first_input = ComplexNumber(1, 2)
-        second_input = ComplexNumber(-5, 10)
-        self.assertEqual(first_input.mul(second_input).real, -25)
-        self.assertEqual(first_input.mul(second_input).imaginary, 0)
+        second_input = ComplexNumber(3, 4)
+        self.assertEqual(first_input.mul(second_input).real, -5)
+        self.assertEqual(first_input.mul(second_input).imaginary, 10)
 
     def test_divide_purely_real_numbers(self):
         input_number = ComplexNumber(1.0, 0.0)
```

Comment:
Can you swap the order the arguments in these methods to match the convention `assertEqual(actual, expected)`?